.github/workflows/deploy.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     # SHA for security hardening. Points at last verified HEAD of main branch.
     uses: primer/.github/.github/workflows/deploy.yml@0cec9b9914f358846163f2428663b58da41028c9
     with:
-      node_version: 12
+      node_version: 14
```

Comment:
I upgraded the `node_version` for compatability with `markdownlint-cli2` node dependency. I saw `ci.yml` already uses `node_version: 14` so assuming this is fine...